client/src/components/TextInputComponent/TextInput.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import "./TextInput.css";
+
+const TextInput = props => <input type="text" placeholder={props.label} />;
```

Comment:
It looks like we need to be able to pass down the `type` as a prop. If you loo at the issue, this component should be able to render a text, email, phone number, or number input. You should rename it to maybe just `Input` and add a prop that lets us control the input type

Reply:
Related to that, can you add a story that demonstrates the different input types from the mock?